
{'date': 'Wed Jul 11 2018 14:18:16.744 GMt-0300 (-03) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 41
    Tabuleiro()
TypeError: __init__() takes 0 positional argument but more were given
'''},
{'date': 'Wed Jul 11 2018 14:37:55.624 GMt-0300 (-03) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 48
    Tabuleiro()
  module <module> line 44
    self.pilha.onclick(move_carta)
AttributeError: 'Elemento' object has no attribute 'onclick'
'''},
{'date': 'Wed Jul 11 2018 14:39:29.384 GMt-0300 (-03) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 48
    Tabuleiro()
  module <module> line 44
    self.pilha.elt.onclick(move_carta)
TypeError: 'NoneType' object is not callable
'''},
{'date': 'Wed Jul 11 2018 16:05:04.882 GMt-0300 (-03) -X- SuPyGirls -X-',
'error': '''
 module <string> line 66
  inicio_x, inicio_y = 
                        ^
SyntaxError: invalid syntax
'''},
{'date': 'Wed Jul 11 2018 16:06:10.335 GMt-0300 (-03) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 75
    Tabuleiro()
  module <module> line 64
    self.casa0.entra(tabelafase1)
AttributeError: 'Tabuleiro' object has no attribute 'casa0'
'''},
{'date': 'Wed Jul 11 2018 16:08:06.26 GMt-0300 (-03) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 76
    Tabuleiro()
  module <module> line 63
    self.tabuleiro = {'0_0':self.tab0, '0_1':self.tab}
AttributeError: 'Tabuleiro' object has no attribute 'tab0'
'''},
{'date': 'Wed Jul 11 2018 17:02:54.497 GMt-0300 (-03) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 80
    Tabuleiro()
  module <module> line 51
    self.lista_de_cartas[carta] = Elemento (carta, tit= carta, style=dict(
AttributeError: 'Tabuleiro' object has no attribute 'lista_de_cartas'
'''},
{'date': 'Wed Jul 11 2018 17:02:55.867 GMt-0300 (-03) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 80
    Tabuleiro()
  module <module> line 51
    self.lista_de_cartas[carta] = Elemento (carta, tit= carta, style=dict(
AttributeError: 'Tabuleiro' object has no attribute 'lista_de_cartas'
'''},
{'date': 'Wed Jul 11 2018 17:37:45.30 GMt-0300 (-03) -X- SuPyGirls -X-',
'error': '''
 module <string> line 57
  a_carta_a_ser_empilhada = Elemento (carta, tit= carta, style=dict(
                                                                                                                                ^
SyntaxError: invalid syntax
'''},